app/controllers/compound_metric_configurations_controller.rb

```diff
@@ -1,72 +1,19 @@
 class CompoundMetricConfigurationsController < BaseMetricConfigurationsController
-  before_action :set_metric_configurations, only: [:new, :edit]
-
-  def create
-    super
-    @compound_metric_configuration.metric.type = "CompoundMetricSnapshot"
-    respond_to do |format|
-      create_and_redir(format)
-    end
-    Rails.cache.delete("#{params[:kalibro_configuration_id].to_i}_tree_metric_configurations")
-  end
-
-  def show
-    update_metric_configuration(@metric_configuration)
-    super
-  end
-
-  def edit
-    @compound_metric_configuration = @metric_configuration
-    @kalibro_configuration_id = params[:kalibro_configuration_id].to_i
-    @compound_metric_configuration.kalibro_configuration_id = @kalibro_configuration_id
-  end
+  METRIC_TYPE = 'CompoundMetricSnapshot'
```

Comment:
Since the behaviour of most of the methods that handle the models is almost exactly equal for the subclasses it seemed like the best solution at the time. The alternative would be copying and pasting some methods in the subclasses while changing one constant, which is quite bad IMO.